Presentation.Web/app/services/dataProcessingRegistrationService.ts

```diff
@@ -231,8 +231,8 @@
 
 
         updateAgreementConcludedAt(dataProcessingRegistrationId: number, dateTime: string) {
-            var date = moment(dateTime, "YYYY-MM-DD");
-            return this.simplePatch(this.getUriWithIdAndSuffix(dataProcessingRegistrationId, "agreement-concluded-at"), date);
+            var formattedDate = moment(dateTime, "DD-MM-YYYY").format("YYYY-MM-DD");
```

Comment:
hvorfor virkede det første gang man sætter det men ikke når man retter det?

Reply:
Er det et problem man har andre steder i KITOS for det er jo ikke første gang man retter datoer

Reply:
det virkede heller ikke helt. Den skrev den forkerte dato ind til serveren. Og det har så været fordi den anden dato du har valgt der har formateringen givet en ikke gyldig dato

Reply:
lad os lige vende den efter mødet